app/templates/main/search.html

```diff
@@ -7,35 +7,55 @@
 
 {% block content %}
 <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-full">
         {{ super() }}
-        <h1 class="govuk-heading-xl">Search</h1>
+        <h2 class="govuk-heading-l">Search for records</h2>
+        <p class="govuk-body-l">Search for digital-born records transfered to The National Archives</p>
 
 
-        <div class="govuk-form-group govuk-!-display-inline">
+        <div class="govuk-form-group">
+          <label class="govuk-label" for="two-thirds">  
+          </label>
+          <input class="govuk-input govuk-!-width-two-thirds" id="two-thirds" name="twoThirds" type="text">
+         <button class="govuk-button" data-module="govuk-button">Search</button>
+
+         
+         
+        <div class="govuk-form-group">
             <fieldset class="govuk-fieldset">
               <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
-                <h1 class="govuk-fieldset__heading">
-                  Filter
-                </h1>
               </legend>
-              <div class="govuk-radios govuk-radios--small" data-module="govuk-radios">
+              
+
+              <div class="govuk-radios govuk-radios--small govuk-radios--inline" data-module="govuk-radios">
+                <p class="govuk-body govuk-link govuk-!-display-inline">
```

Comment:
I think we should move this p block to after the radio buttons, there seems to be some css that we might need to add to get everything lined up. Looks good so far.